client/src/localization/languages/Sv.tsx

```diff
@@ -0,0 +1,226 @@
+    com_endpoint_config_key_chatgpt: 'För att få din åtkomstnyckel för ChatGPT "Gratisversion", logga in på', // To get your Access token For ChatGPT 'Free Version', login to
+    com_endpoint_config_key_chatgpt_then_visit: 'sedan besök', // then visit
+    com_endpoint_config_key_chatgpt_copy_token: 'Kopiera åtkomstnyckel.', // Copy access token.
+    com_endpoint_config_key_google_need_to: 'Du behöver', // You need to
+    com_endpoint_config_key_google_vertex_ai: 'Aktivera Vertex AI', // Enable Vertex AI
+    com_endpoint_config_key_google_vertex_api: 'API på Google Cloud, sedan', // API on Google Cloud, then
+    com_endpoint_config_key_google_service_account: 'Skapa ett tjänstekonto', // Create a Service Account
+    com_endpoint_config_key_google_vertex_api_role:
+      'Se till att klicka på "Skapa och fortsätt" för att ge åtminstone rollen "Vertex AI-användare". Skapa slutligen en JSON-nyckel att importera här.', // Make sure to click 'Create and Continue' to give at least the 'Vertex AI User' role. Lastly, create a JSON key to import here.
+    com_nav_export_filename: 'Filnamn', // Filename
+    com_nav_export_filename_placeholder: 'Ange filnamnet', // Set the filename
+    com_nav_export_type: 'Typ', // Type
+    com_nav_export_include_endpoint_options: 'Inkludera slutpunktsalternativ', // Include endpoint options
+    com_nav_enabled: 'Aktiverad', // Enabled
+    com_nav_not_supported: 'Stöds ej', // Not Supported
+    com_nav_export_all_message_branches: 'Exportera alla grenar för meddelanden', // Export all message branches
+    com_nav_export_recursive_or_sequential: 'Rekursiv eller sekventiell?', // Recursive or sequential?
+    com_nav_export_recursive: 'Rekursiv', // Recursive
+    com_nav_export_conversation: 'Exportera konversation', // Export conversation
+    com_nav_theme: 'Tema', // Theme
+    com_nav_theme_system: 'System', // System
+    com_nav_theme_dark: 'Mörkt', // Dark
+    com_nav_theme_light: 'Ljust', // Light
+    com_nav_clear_all_chats: 'Rensa alla chattar', // Clear all chats
+    com_nav_confirm_clear: 'Bekräfta rensning', // Confirm Clear
+    com_nav_close_sidebar: 'Stäng sidofält', // Close sidebar
+    com_nav_open_sidebar: 'Öppna sidofält', // Open sidebar
+    com_nav_log_out: 'Logga ut', // Log out
+    com_nav_user: 'ANVÄNDARE', // USER
+    com_nav_clear_conversation: 'Rensa konversationer', // Clear conversations
+    com_nav_clear_conversation_confirm_message:
+        'Är du säker på att du vill rensa alla konversationer? Detta går inte att ångra.', // Are you sure you want to clear all conversations? This is irreversible.
+    com_nav_help_faq: 'Hjälp & Vanliga frågor', // Help & FAQ
+    com_nav_settings: 'Inställningar', // Settings
+    com_nav_search_placeholder: 'Sök meddelanden', // Search messages
+    com_nav_setting_general: 'Allmänt', // General
+    com_nav_setting_data: 'Datakontroller', // Data controls
+    com_nav_language: 'Språk', // Language
```

Comment:
Can you please remove "com_nav_language"?

Reply:
Thank you, missed that! It has now been removed